chore(gcp): Update usage documentation (#1955)
cloudquery/cloudquery: plugins/source/gcp/README.md

# GCP Source Plugin

> GCP Source plugin for CloudQuery extract configuration from varietly of GCP APIs and loads them to any of the supported CloudQuery Destination plugins.

Reviewer: ```suggestion
The GCP Source plugin for CloudQuery extracts configuration from a variety of GCP APIs and loads them into any supported CloudQuery destinations (e.g. PostgreSQL).
```

## Links

- [Tables](./docs/tables/README.md)
- [Homepage](https://cloudquery.io)

> - [Documentation](https://docs.cloudquery.io)

Author: I see. let's keep the `cloudquery.io` for now and check if we can change the permanent one to be without `www`

> - Tables: [./docs/tables/README.md](./docs/tables/README.md)
> - Homepage: [https://cloudquery.io](https://cloudquery.io)
> - Documentation: [https://docs.cloudquery.io](https://docs.cloudquery.io)
> - Discord: [https://cloudquery.io/discord](https://cloudquery.io/discord)

Reviewer 2: ```suggestion
- [Tables](./docs/tables/README.md)
- [Homepage](https://www.cloudquery.io)
- [Documentation](https://www.cloudquery.io/docs)
- [Discord](https://www.cloudquery.io/discord)
```

Might be nicer not to have file paths as link names.
This is how it looks before the suggestion
![image](https://user-images.githubusercontent.com/26760571/191711108-243b1291-820d-49d7-b962-6227e049c18c.png)

Author: I agree on the links but don't agree on the `www` as without it those are the canonical links

Author: Fixed the links

## Install

```bash
cloudquery gen source gcp
```

## Libraries in Use

- https://cloud.google.com/go/docs/reference

> - https://github.com/googleapis/google-api-go-client

Reviewer 2: This seems to belong in the contributing guide, not sure it belongs so early in the README.md

Author: I think it can also be useful for users to understand where is the original documentation (but also for devs)
